Content.Server/Objectives/Conditions/StealCondition.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using Content.Server.Mobs;
+using Content.Server.Objectives.Interfaces;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.Serialization;
+using Robust.Shared.Utility;
+
+namespace Content.Server.Objectives.Conditions
+{
+    public class StealCondition : IObjectiveCondition
+    {
+        public string PrototypeID { get; private set; }
+
+        public void ExposeData(ObjectSerializer serializer)
+        {
+            serializer.DataField(this, x => x.PrototypeID, "prototype", "");
+        }
+
+        public string GetTitle() => $"Steal prototype {PrototypeID}";
+
+        public string GetDescription() => $"We need you to steal prototype {PrototypeID}. Please dont get caught.";
```

Comment:
I'd make it just "... . Don't get caught."

Reply:
im too polite to write syndicate orders help pls